cluster-provision/images/vm-image-builder/publish-multiarch-containerdisk.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+set -exuo pipefail
+
+SCRIPT_PATH=$(dirname "$(realpath "$0")")
+archs=(aarch64 x86_64)
```

Comment:
Again, please use Go-style architecture names here, converting only if and when necessary.